package.json

```diff
@@ -26,6 +26,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "react-slick": "^0.28.1",
+    "reading-time": "^1.3.0",
```

Comment:
Can we use this util function instead? https://github.com/openebs/website/blob/3481431ceac30c64ca3b1dc68a1550665d30b163/src/utils/readingTime.ts

Reply:
sure, will use this util instead. Thanks

Reply:
This npm package is not needed.

Reply:
sure